pumps/mongo_aggregate.go

```diff
@@ -416,3 +460,25 @@ func (m *MongoAggregatePump) collectionExists(name string) (bool, error) {
 func (m *MongoAggregatePump) WriteUptimeData(data []interface{}) {
 	m.log.Warning("Mongo Aggregate should not be writing uptime data!")
 }
+
+// getLastDocumentTimestamp will return the timestamp of the last document in the collection
+func getLastDocumentTimestamp(session *mgo.Session, collectionName string) (time.Time, error) {
+	var doc bson.M
+	err := session.DB("").C(collectionName).Find(nil).Sort("-$natural").One(&doc)
```

Comment:
will this work in all the databases?

Reply:
@mativm02 can you check @sredxny question?

Reply:
It's currently working on Cosmos, DocDB and Standard Mongo!